docs/_releases/v2.0.0-pre.4/matchers.md

```diff
@@ -112,6 +112,26 @@ Requires the value to be a `Function`.
 Requires the value to be an `Array`.
 
 
+#### `sinon.match.array.deepEquals(iterable)`
+
+Requires an iterable value to be deep equal another one.
```

Comment:
I understand the benefits and usage. I'm just saying when you document it, it looks weird/out of place. I would expect an *array* matcher to throw/error if I didn't pass it an array honestly. I was considering an `iterable` matcher. I'd wait to see what other thing about it though.